Podfile

```diff
@@ -11,14 +11,14 @@ workspace 'WordPress.xcworkspace'
 ##
 def wordpress_shared
     ## for production:
-    pod 'WordPressShared', '1.8.10'
+    pod 'WordPressShared', '1.8.12-beta.2'
```

Comment:
It seems odd that we have a beta version here?

Reply:
Yep! It is. We had a very late fix merged on Friday that was approved this way. A beta 3 was already released, so the clean way would had been to branch from 1.8.11, reapply the patch and release a new version, but this way we had very little testing time, so decided to move this ahead with the beta for this time.